basics.js

```diff
@@ -3,20 +3,23 @@
 // prefixed with the text "I have a " (so "I have a cat", "I have a dog", etc.)
 
 var pets = ['cat', 'dog', 'wombat'];
-
+for (var prop in pets){
```

Comment:
I might not call each element a `prop`, as that's kinda unclear. I'd say `for (pet in pets)`. Only use for-in if order of the execution isn't important.